src/reactpy/core/vdom.py

```diff
@@ -23,12 +24,16 @@
     EventHandlerDict,
     EventHandlerType,
     ImportSourceDict,
+    JavaScript,
+    JSExecutableDict,
     VdomAttributes,
     VdomChildren,
     VdomDict,
     VdomJson,
 )
 
+EVENT_ATTRIBUTE_PATTERN = re.compile(r"^on\w+")
```

Comment:
This can be reverted to `^on[A-Z]` once we figure out why `string_to_reactpy` isn't properly mutating `onclick` to `onClick`.

Reply:
Done.